onnxruntime/core/providers/iree/compiler/jit_compiler.cc

```diff
@@ -208,12 +208,16 @@ common::Status CompilerInvocation::ImportSubgraph(const onnxruntime::GraphViewer
   return common::Status::OK();
 }
 
-common::Status CompilerInvocation::CompileAndOutputVMFB(iree_compiler_output_t* output) {
+common::Status CompilerInvocation::CompileAndOutputVMFB(iree_compiler_output_t* output, std::string save_to) {
   // Main compilation.
   if (!ireeCompilerInvocationPipeline(inv, IREE_COMPILER_PIPELINE_STD)) {
     return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_GRAPH, "IREE compilation error.", ConsumeDiagnostics());
   }
 
+  // Attach the compiled output to a file.
+  save_to.append("compiled_model.vmfb");
```

Comment:
It might be a good idea to use something other than a string. See the `filesystem` library, for example.